.github/workflows/fast_testing.yaml

```diff
@@ -39,19 +40,58 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Install tarantool ${{ matrix.tarantool }}
+        if: startsWith(matrix.tarantool, 'debug') != true
         uses: tarantool/setup-tarantool@v1
         with:
           tarantool-version: ${{ matrix.tarantool }}
 
+      - name: Create version postfix for Tarantool
+        if: startsWith(matrix.tarantool, 'debug')
+        run: |
+          branch=$(echo ${{ matrix.tarantool }} | cut -d- -f2)
+          commit_hash=$(git ls-remote https://github.com/tarantool/tarantool.git --branch ${branch} | head -c 8)
+          echo "version_postfix=-${commit_hash}" >> $GITHUB_ENV
+        shell: bash
+
+      - name: Cache tarantool build
+        if: startsWith(matrix.tarantool, 'debug')
+        id: cache-tnt-debug
+        uses: actions/cache@v3
+        with:
+          path: ~/tnt-debug/
+          key: cache-tnt-${{ matrix.tarantool }}${{ env.version_postfix }}
+
+      - name: Build tarantool ${{ matrix.tarantool }}
+        if: startsWith(matrix.tarantool, 'debug') && steps.cache-tnt-debug.outputs.cache-hit != 'true'
+        run: |
+          sudo apt-get -y install git build-essential cmake make zlib1g-dev \
+            libreadline-dev libncurses5-dev libssl-dev \
+            libunwind-dev libicu-dev python3 python3-yaml \
```

Comment:
For master you don't need libunwind AFAIK.

It's submodule now https://github.com/tarantool/tarantool/commit/7dc9fe44a5aebc2b44c240221d2a71cd6d97b6b0

Reply:
Thank you for the note! I have added the ability to use a specific branch from the Tarantool's repository.

Anyway, let's just copy the list from the documentation. It's too hard to analyze it. It does not give a lot of overhead.